tech_reports/LLMs/llms.md

```diff
@@ -43,8 +43,154 @@ Authors:
   - Iterative update system
   - When to use our fused op
 ### 2.3 Norm
-  - Replicated layernorm vs distributed layernorm
-    - Layernorm/rmsnorm weights in row major / wrapped around tile size trick
+
+Normalization is a critical operation in Large Language Models (LLMs), ensuring stable training and efficient inference. Two widely adopted normalization techniques in modern LLMs, **LayerNorm** and **RMSNorm**, are fully supported in TT-NN.
+
+#### Implementations of Normalization Operations
+
+TT-NN includes two primary implementations of normalization operations to handle diverse activation layouts efficiently:
+
+1. **Non-Distributed Norm**
+2. **Distributed Norm**
+
+
+#### 1. Non-Distributed Norm
+
+This implementation supports both sharded and interleaved inputs. It is employed in the following scenarios:
```

Comment:
What does it mean to be non-distributed? Needs to be mentioned

Reply:
Thanks, addressed it!